dist/images/Dockerfile.ubuntu

```diff
@@ -0,0 +1,51 @@
+#
+# The standard name for this image is ovn-kube
```

Comment:
Can you change the comment to reflect what is really built finally in the Makefile? 'ovn-kube-u' that is.

Reply:
Done